test/test_hyperparameters.py

```diff
@@ -328,6 +326,177 @@ def test_normalfloat_to_integer(self):
         f2_actual = f1.to_integer()
         self.assertEqual(f2_expected, f2_actual)
 
+    def test_betafloat(self):
+        # TODO test non-equality
+        f1 = BetaFloatHyperparameter("param", lower=-2.0, upper=2.0, alpha=3.0, beta=1.0)
+        f1_ = BetaFloatHyperparameter("param", lower=-2, upper=2, alpha=3, beta=1)
+        self.assertEqual(f1, f1_)
+        self.assertEqual(
+            "param, Type: BetaFloat, Alpha: 3.0 Beta: 1.0, Range: [-2.0, 2.0], Default: 2.0",
+            str(f1_))
+
+        # test parameters that do not create a legit beta distribution
+        with self.assertRaises(ValueError):
```

Comment:
Does this really test both hyperparameters?

Reply:
Oh, right. The assertion occur once it gets the error from the first one, right? Good catch, then.

Reply:
Fixed this.